modelskill/metrics.py

```diff
@@ -589,12 +590,10 @@ def peak_ratio(
         )
         peaks = data[peak_index]
         peaks_sorted = peaks.sort_values(ascending=False)
-        found_peaks.append(
-            peaks_sorted[0 : max(1, min(round(AAP_ * N_years), np.sum(peaks)))]
-        )
+        found_peaks.append(peaks_sorted)
     found_peaks_obs = found_peaks[0]
     found_peaks_mod = found_peaks[1]
-
+    top_n_peaks = max(1, min(round(AAP_ * N_years), np.sum(peaks)))
```

Comment:
AAP_ is calculated for both obs and model in the loop above. Only the last value is used (i.e. model). I would avoid the loop and do the calculation in two calls for obs and model to make it explicit.

```python
peak_index, _ = _find_peaks(obs, ...
peak_index, AAP_ = _find_peaks(model, ...
```

Reply:
ok let me rewrite this section

Reply:
Done as suggested